### PR TITLE
Updates linter to latest version

### PR DESCRIPTION
**Pull request by kathyoleary**
In order to keep the linter up to date
As an LMHT coder,
I want the latest version of the linter installed so we are always using the latest rules on ESlint

Acceptance Criteria:
Install latest version of linter 2.13.1
Update the rules with any requests that could not be accomplished in other PR's because we had previous version.

Given any LMHT coder,
I want to be using the latest greatest linter files so everything works smoothly based on the latest data.

### CODE REVIEW

**Comment by rylnd** [comment]
@kathyoleary please indicate that a PR is still in progress (and not yet meant to be reviewed) by putting "WIP" or 🚧  in the title/description.

File: .eslintrc.js [hunk 1]
```diff
@@ -23,6 +23,9 @@ module.exports = {
       "always"
     ],
     "no-multiple-empty-lines": [2, {"max": 2}],
+    "space-in-parens": [2, "never"],
+    "keyword-spacing": [2, { "before": true}],
```
Comment (rylnd) [inline .eslintrc.js]: I believe you can (and should) simplify these two lines to:

``` json
"keyword-spacing": [2, {"before": true, "after": true}],
```

File: .eslintrc.js [hunk 2]
```diff
@@ -23,6 +23,9 @@ module.exports = {
...
+    "keyword-spacing": [2, { "after": true}],
```
Comment (rylnd) [inline .eslintrc.js]: Please also add the [space-before-blocks](http://eslint.org/docs/rules/space-before-blocks) rule as mentioned in #180.

**Comment by rylnd** [comment]
@kathyoleary with this upgrade you _can_ now replace the `2`s with `"error"` as we discussed in #71. Up to you whether that's worthwhile or not.

**Comment by kathyoleary** [comment]
Was going to, did it while in conference with Jim, so we were just throwing it through as fast as possible and then I was coming out here to finish updating/fixing.  It's all good now including the rest of the updates to get it current with new version.

**Comment by rylnd** [comment]
👍

**Comment by bigtiger** [comment]
👍

**Pull request merged**